lib/rack/twilio_webhook_authentication.rb

```diff
@@ -17,18 +17,20 @@ module Rack
   # doesn't validate then the middleware responds immediately with a 403 status.
 
   class TwilioWebhookAuthentication
-    def initialize(app, auth_token, *paths)
+    def initialize(app, auth_token, *paths, &block)
```

Comment:
maybe one more nit: rename this “block” to “auth_block” or something so its purpose is obvious :)